channels/1ds-post-js/src/PostChannel.ts

```diff
@@ -259,6 +274,41 @@ export class PostChannel extends BaseTelemetryPlugin implements IChannelControls
                 };
             }
 
+            function _batch(arr: string[]) {
+                let rlt = EMPTY_STR;
+                
+                if (arr && arr.length) {
+                    arrForEach(arr, (item) => {
+                        if (rlt) {
+                            rlt += "\n";
+                        }
+                        rlt += item;
+
+                    });
+                }
+                return rlt;
+
+            }
+
+            function _serialize(event: ITelemetryItem) {
+
+                let rlt = EMPTY_STR;
+                try {
+                    _cleanEvent(event);
+                    if (_overrideInstrumentationKey) {
```

Comment:
Question for review: when is this called (if I haven't worked it out already) as do we really want / need to override the instrumentation key.

ie. We don't want to override any previously save batches (because they may actually belong to someone else)